CHANGELOG.md

```diff
@@ -1,3 +1,338 @@
+  kubectl delete pvc data-alertmanager-prometheus-operator-alertmanager-0 -n monitoring
+  ```
+
+  You can choose to remove all your existing CRDs (ServiceMonitors, PodMonitors, etc.) if you want to.
+
+3. Remove current `spec.claimRef` values to change the PV's status from Released to Available.
+
+  ```bash
+  kubectl patch pv --type json -p='[{"op": "remove", "path": "/spec/claimRef"}]' $(kubectl get pv -o jsonpath='{.items[?(@.spec.claimRef.name=="data-prometheus-prometheus-operator-prometheus-0")].metadata.name}')
+
+  kubectl patch pv --type json -p='[{"op": "remove", "path": "/spec/claimRef"}]' $(kubectl get pv -o jsonpath='{.items[?(@.spec.claimRef.name=="data-alertmanager-prometheus-operator-alertmanager-0")].metadata.name}')
+  ```
+
+  > **Note:** To execute the above command, the user must have a cluster wide permission.
+
+4. Make sure that the `prometheus-operator`'s `storage_class` and `prometheus.storage_size` are unchanged during the upgrade process.
+
+5. Proceed to a fresh `prometheus-operator` component installation. The new release should now re-attach your previously released PV with its content.
+
+```
+lokoctl component apply prometheus-operator
+```
+
+6. If monitoring was enabled for `rook`, `contour`, `metallb` components, then update the said components as well after the cluster is updated.
+
+#### Cluster update steps
+
+> **NOTE**: Updating multiple Lokomotive versions at a time is not supported. If your cluster is running a version older than `v0.5.0`, update to `v0.5.0` first and only then proceed with the update to `v0.6.0`.
+
+Please perform the following manual steps in your cluster configuration directory.
+
+1. Download the release bundle.
+
+  ```bash
+  curl -LO https://github.com/kinvolk/lokomotive/archive/v0.6.0.tar.gz
+  tar -xvzf v0.6.0.tar.gz
+  ```
+
+2. Install the Packet CCM.
+
+  If you are running Lokomotive on Equinix Metal (formerly Packet), then install Packet CCM. Export your Packet cluster's project ID and API Key.
+
+  ```bash
+  export PACKET_AUTH_TOKEN=""
+  export PACKET_PROJECT_ID=""
+
+  echo "apiKey: $PACKET_AUTH_TOKEN
+projectID: $PACKET_PROJECT_ID" > /tmp/ccm-values.yaml
+
+  helm install packet-ccm --namespace kube-system --values=/tmp/ccm-values.yaml ./lokomotive-0.6.0/assets/charts/control-plane/packet-ccm/
+  ```
+
+3. Update node config.
+
+  This script shipped with release tarball will add permanent MetalLB labels and kubelet config to use CCM.
+
+  > **NOTE**: Please edit this script to disable updating certain nodes. Modify `update_other_nodes` function as required.
+
+  ```bash
+  ./lokomotive-0.6.0/scripts/update/0.5-0.6/update.sh false
+  ```
+
+4. Reload the new config for self-hosted kubelet.
+
+  ```bash
+  kubectl -n kube-system rollout restart ds kubelet
+  ```
+
+  If the above command failed, probably you are not running self-hosted kubelet, which is fine because the script in step 3 already updated the host kubelet.
```

Comment:
I don't like this "probably" there, how about this?

```suggestion
4. If you're using the self-hosted kubelet, reload its config.

  ```bash
  kubectl -n kube-system rollout restart ds kubelet
  ```

  If you're unsure you can run the command as it's harmless if you're not using the self-hosted kubelet.
```